demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

```diff
@@ -16,12 +15,3 @@
         <div>Courier: @Model.ShippingCourier</div>
     </div>
 </div>
-
-@*<div class="panel panel-default">
-    <div class="panel-heading">
-        <h3 class="panel-title">Shipment information</h3>
-    </div>
-    <div class="panel-body">
-
-    </div>
-</div>*@
```

Comment:
@mauroservienti this file belongs to the existing solution, did this change creep into this PR by accident?

(Same with the .suo file above.)